libraries/core_libs/storage/src/storage.cpp

```diff
@@ -420,6 +420,10 @@ void DbStorage::clearPeriodDataHistory(uint64_t period) {
   db_->DeleteRange(write_options_, handle(Columns::period_data), toSlice(start), toSlice(period));
 }
 
+void DbStorage::UpdateCertVotesInPeriodData(const SyncBlock& sync_block) {
```

Comment:
you are not updating cert votes in db here, you are saving/overriding whole sync block data

Reply:
Since we only save cert votes in sync block object in period_data column in DB, we don't save cert votes separately.
I think this is the only way to overriding whole sync block data in DB.
If you know how to update without overriding, could you please teach me? Thanks!

Reply:
I mean - rename the method because you are not updating cert votes, you are overring/resaving the whole sync  block in db

Reply:
I will change function name to overridePeriodData